MANIFEST.in

```diff
@@ -1,4 +1,5 @@
-prune .
```

Comment:
The logs when building the source distribution showed that this didn't seem to be doing anything. I saw:
```
reading manifest template 'MANIFEST.in'
no previously-included directories found matching '.'
```

I switched to using `include` rather than the recursive form just to be a little more explicit.